### DISCUSSION TITLE
[Question] included CKEditor source

### DISCUSSION THREAD

**Discussion by tom-programming** [TriliumNext/Trilium#6324] [Q&A] [answered]
Hi,
in the repository the CKEditor library is included as a minified js file. Some settings, like the math delimiters seem buried in this file (e.g. line 6, col 1184672). Is there a way to include a human-readable version of CKEditor and compile it into the form that Trilium requires?

**Comment by Nriver** [comment, marked as answer]
The code is here https://github.com/zadam/trilium-ckeditor5
